xcode/Subconscious/Shared/Components/Common/EntryList/EntryListView.swift

```diff
@@ -35,6 +37,7 @@ struct EntryListView: View {
                                 onLink: onLink
                             )
                         }
+                        .id(idx)
```

Comment:
This is likely to cause issues, as ids are meant to be stable identifiers tied to records. If the record changed but had same index, or if the same record changed index, we might see view invalidation bugs.

Should replace with a stable record ID of some sort.

Reply:
Good note, I've ditched this approach and inserted an `EmptyView` with a stable `.id()` at the top instead. This is easier to read as well imo, so win-win.